library/nats

```diff
@@ -3,31 +3,31 @@ Maintainers: Derek Collison <derek@synadia.com> (@derekcollison),
              Waldemar Salinas <wally@synadia.com> (@wallyqs),
              Jaime Piña <jaime@synadia.com> (@variadico)
 GitRepo: https://github.com/nats-io/nats-docker.git
```

Comment:
To fix the (new) CI failure (#11071), I think you'll want to add the following here:

```suggestion
GitRepo: https://github.com/nats-io/nats-docker.git
GitFetch: refs/heads/main
```